tv_apps/fling-player/js/connector.js

```diff
@@ -71,13 +71,33 @@
     this._connection.onstatechange = this.onConnectionStateChange.bind(this);
   };
 
+  proto.isConnected = function () {
+    return this._isInit && this._isInitConnection;
+  };
+
+  /**
+   * @param {strin} type 'sent' or 'received'
+   * @return {number} If no sent or no received or unknown type, would return -1
+   */
+  proto.getLastMsgSeq = function (type) {
```

Comment:
Since the original message doesn't contain sequence number, is it appropriate to append a public method for just testing use? Can we skip the sequence testing?